lmfit/minimizer.py

```diff
@@ -40,6 +40,7 @@
 try:
     import emcee as emcee
     HAS_EMCEE = True
+    EMCEE_MAJOR_VERSION_CHECK = int(emcee.__version__[0]) >= 3
```

Comment:
I am not a huge fan of the variable name, you're checking if emcee is version 3 or not - what about ```EMCEE_MAJOR_VERSION_3``` instead?

Reply:
changed the usage emcee version checking to `EMCEE_VERSION`, as specified by @newville